ordered_associative_task/student/include/impl.h

```diff
@@ -15,4 +15,21 @@
  *
  * @return 1-5
  */
-size_t convertMarkTo5PointSystem(size_t markIn100Point);
+size_t convertMarkTo5PointSystem(size_t markIn100Point) {
+    int maxMark = 5;
+    std::map<int, int> pointSystem {
+        {29, 1},
+        {49, 2},
+        {69, 3},
+        {89, 4},
+        {90, maxMark}
+    };
+
+    auto reultIter = pointSystem.lower_bound(markIn100Point);
+
+    if (reultIter == pointSystem.cend()) {
+        return maxMark;
```

Comment:
please solve this task only with map, without special values

Reply:
done